### ISSUE TITLE
Firefox Port / Inconsistencies

### ISSUE THREAD

**Issue by phantyx** [RyanSzczepanski/Old-Fashion-Description#15]
This is godawful code, as far as i can tell. I've never worked in JS before. Regardless, it's semi-functional on release. Hoping you can refine this to be more consistent for a FF port. (I forked your project). Also, I didn't want to submit it as an extension on mozilla's workshop thing since copyright.

**Comment by RyanSzczepanski** [comment]
Yeah, this project may not be ideally designed, this was my first every shot at making an extension.  The extension was running fine as I used it but the major issue is that YouTube rolls out updates to different people at different times so I can't account for everyone that uses this extension.  I also can no longer continue development on this extension because my version of YouTube no longer contains the changed description.  If you are able to make a better version of this extension I encourage you to post it on the Mozilla Addon store because the whole reason of me creating the addon was that I was annoyed at the YouTube changes and I wanted to fix it for others who felt the same.  You should be in the clear to upload it [Mozilla Public License](https://www.mozilla.org/en-US/MPL/2.0/) as long as its not an unedited copy and paste, and even if it is I'm not gonna do anything about it.  With that being said, Have a good day.
